lib/administrate/search.rb

```diff
@@ -38,6 +47,42 @@ def attribute_types
       resolver.dashboard_class::ATTRIBUTE_TYPES
     end
 
-    attr_reader :resolver, :term
+    def search_scope(term)
+      if term && (/scope:(?<possible_scope>.+)/i =~ term)
+        possible_scope if resource_class.respond_to?(possible_scope) &&
+                          valid_scope?(possible_scope)
```

Comment:
Use 4 (not 18) spaces for indenting a condition in an `if` statement spanning multiple lines.